### ISSUE TITLE
Can't make it work

### ISSUE THREAD

**Issue by altrusl** [emersonbottero/vitepress-plugin-search#79]
Hello Emerson

I am trying to set it up as described in your readme

But first of all - what is _flexSearchIndexOptions_? They are not in the `vitepress-plugin-search` package

I installed flexsearch and vitepress-plugin-search, it runs in dev mode and builds correctly, but no search fields appear
No errors in the console, no logs
Should there be some special configuration for your plugin?
I use `pnpm vitepress dev docs` command to run the site

PNPM, Vite 4.4.5, Vue 3.3.4, vitepress 1.0.0-rc.4, typescript 5.0.2
vitepress-plugin-search: 1.0.4-alpha.20

**Comment by emersonbottero** [comment]
1 - [flexSearchIndexOptions](https://github.com/nextapps-de/flexsearch#options)
2 - Working example https://github.com/emersonbottero/vitepress-demo2 

you should see an 
🔎 Indexing...
🔎 Done.

in the console logs if correctly installed.

**Comment by emersonbottero** [comment]
@altrusl did it worked?

**Comment by manawasp** [comment]
Hello @emersonbottero ,

I have the same case, I see nothing (no log, no error, no search box). I did a poc months ago and it was working then after some lib updated i noticed the search box disappeared and I didn't dive further at the time. I gave a new try today and still see nothing.

I tried to setup https://github.com/emersonbottero/vitepress-demo2 to see if it is working and I have a blank page with a console error (but I believe not related to the current issue). 

My setup:

```
$ nvm version
v20.3.0
$ pnpm --version
8.12.0
```

I forced also a rollback on `vitepress`:

```
devDependencies:
- vitepress 1.0.0-rc.36
+ vitepress 1.0.0-rc.4 (1.0.0-rc.36 is available)
```

Didn't change the result.

If you have some idea how to dive deeper on this issue will be a pleasure to help !
Thanks :pray:
